app/src/features/projects/view/components/FundingSource.tsx

```diff
@@ -0,0 +1,127 @@
+  const {
+    projectWithDetailsData: { funding }
+  } = props;
+
+  const classes = useStyles();
+
+  return (
+    <>
+      <Grid container spacing={3}>
+        <Grid container item xs={12} spacing={3} justify="space-between" alignItems="center">
+          <Grid item>
+            <Typography variant="h3">Funding Sources</Typography>
+          </Grid>
+          <Grid item>
+            <Button
+              variant="outlined"
+              component="label"
+              size="medium"
+              color="primary"
+              className={classes.addButton && classes.heading}>
+              Add Funding Source
+            </Button>
+          </Grid>
+        </Grid>
+        {funding.fundingAgencies.map((item: any) => (
+          <Fragment key={item.agency_id}>
+            <Grid container item>
+              <Divider className={classes.topBorder} />
+            </Grid>
+            <Grid container item spacing={3} xs={12} justify="space-between" alignItems="center">
+              <Grid item xs={12} sm={6} md={4}>
+                <Box>
+                  <Typography className={classes.heading}>{item.agency_name}</Typography>
+                </Box>
+              </Grid>
+              <Grid item>
+                <IconButton title="Edit General Information" aria-label="Edit General Information">
+                  <Typography variant="caption">
+                    <Edit fontSize="inherit" /> EDIT
+                  </Typography>
+                </IconButton>
+              </Grid>
+            </Grid>
+            <Grid container item spacing={3} xs={12}>
+              <Grid item xs={12} sm={6} md={4}>
+                <Box color="text.disabled">
+                  <Typography variant="caption">Agency Project ID</Typography>
+                </Box>
+                <Box>
+                  <Typography style={{ wordBreak: 'break-all' }} variant="subtitle1">
+                    {item.agency_id}
+                  </Typography>
+                </Box>
+              </Grid>
+              <Grid item xs={12} sm={6} md={4}>
+                <Box color="text.disabled">
+                  <Typography variant="caption">Funding Amount</Typography>
+                </Box>
+                <Box>
+                  <Typography style={{ wordBreak: 'break-all' }} variant="subtitle1">
+                    {item.funding_amount}
+                  </Typography>
+                </Box>
+              </Grid>
+              <Grid item xs={12} sm={6} md={4}>
+                <Box color="text.disabled">
+                  <Typography variant="caption">Funding Dates</Typography>
+                </Box>
+                <Box>
+                  <Typography style={{ wordBreak: 'break-all' }} variant="subtitle1">
+                    {getFormattedDate(DATE_FORMAT.ShortDateFormatMonthFirst, item.start_date)} -{' '}
+                    {getFormattedDate(DATE_FORMAT.ShortDateFormatMonthFirst, item.end_date)}
```

Comment:
Probably yields the same result, but there is also a `getFormattedDateRangeString` function in `utils/Utils` that handles this case.